package.json

```diff
@@ -26,7 +26,7 @@
     "clean": "rimraf packages/**/.coverage packages/**/dist packages/**/types",
     "prebuild": "yarn run clean",
     "build": "node ./tooling/index.js build && yarn run build:tsc",
-    "build:tsc": "yarn run tsc -p ./config/tsconfig.typecheck.json --noEmit && yarn workspace @webex/plugin-meetings run build",
+    "build:tsc": "yarn run tsc -p ./config/tsconfig.typecheck.json --noEmit && yarn workspace @webex/internal-plugin-metrics run build && yarn workspace @webex/plugin-meetings run build",
```

Comment:
why do we need this now? plugin-meetings was already using internal-plugin-metrics earlier and it worked without this, right? Is there maybe some better way in yarn workspaces to specify this dependency?

Reply:
this is needed to make types working (when we import things from the internal-plugin-meetings)

Reply:
ok, I guess it's fine for now, but feels a bit hacky, @arun3528 does someone from SDK team maybe know how this should be done properly in yarn workspace?